### PR TITLE
[RFC] alsa_setting: remove the duplicated scripts of amixer setting

### PR DESCRIPTION
**Pull request by keqiaozhang**
Signed-off-by: Keqiao Zhang <keqiao.zhang@intel.com>

### CODE REVIEW

**Comment by keqiaozhang** [comment]
Test PR, not ready for review.

**Comment by marc-hb** [comment]
> Test PR, not ready for review.

Next time (too late here now) use a github draft: they don't notify anyone.

Drafts are great, I always start all my PRs as drafts. There are only advantages, zero drawback.

**Comment by fredoh9** [comment]
Removing duplicated settings are good idea.
But mapping is not intuitive. for example, ADLP_RVP_SDW.sh -> RT711_SDW.sh
Can we keep the platform name file and call "RT711_SDW.sh" internally?

**Comment by keqiaozhang** [comment]
> Can we keep the platform name file and call "RT711_SDW.sh" internally?

Do you mean to keep duplicates while using internal mappings?

File: case-lib/lib.sh [hunk 1]
```diff
@@ -442,7 +442,23 @@ set_alsa_settings()
...
-        *)
+        TGLH_0A5E_SDW | CML_SKU0983_SDW | TGLU_SKU0A3E_SDW | TGLU_RVP_SDW | ADLP_RVP_SDW)
+	    # common RT711 SDW setting
+	    "$SCRIPT_HOME"/alsa_settings/RT711_SDW.sh
+	;;
+        ADLP_RVP_HDA | CML_RVP_HDA | CML_SKU0955_HDA | TGLH_RVP_HDA | TGLH_0A70_HDA | TGLU_SKU09FB_HDA | TGLU_RVP_HDA | JSL_RVP_HDA | ICL_RVP_HDA)
+	    # common RT711 HDA setting
+	    "$SCRIPT_HOME"/alsa_settings/GENERIC_HDA.sh
+	;;
+        TGLU_SKU0A32_SDCA | TGLU_SKU0A32_SDCA)
```
Comment (greg-intel) [inline case-lib/lib.sh:453]: Is TGLU_SKU0A32_SDCA listed twice here?

**Pull request closed without merging**